File: example/android/device/src/main/java/com/qc/device/model/Calendar.kt
package com.qc.device.model

import com.google.gson.annotations.SerializedName

data class Calendar(
    @SerializedName("id") val id: Long,
    @SerializedName("event_title") val eventTitle: String,
    @SerializedName("description") val description: String,
    @SerializedName("start_time") val startTime: String,
    @SerializedName("end_time") val endTime: String,
)

File: ios/Classes/model/Calendar.swift
//
//  Calendar.swift
//  flutter_device
//
//  Created by atlands on 2023/8/3.
//

import Foundation

struct Calendar: Codable {
    let id: String?
    let eventTitle: String?
    let description: String?
    let startTime: String?
    let endTime: String?
    let createdAt: String?
    
    enum CodingKeys: String, CodingKey {
        case id
        case eventTitle = "event_title"
        case description
        case startTime = "start_time"
        case endTime = "end_time"
        case createdAt
    }
}
